test/models/movie_test.rb

```diff
@@ -0,0 +1,63 @@
+require "test_helper"
+
+describe Movie do
+  describe 'validations' do
+    before do
+      @movie = Movie.new(title: "titanic", overview: "long", release_date: "1990-03-20", inventory: 10) 
+    end
+    
+    it 'can make a movie with a title' do
+      assert @movie.valid?
```

Comment:
These kinds of assertions work, but they're not in the style we're doing at Ada.  So I'd suggest conforming to the spec-style testing.

```suggestion
      expect(@movie.valid?).must_equal true
```